### ISSUE TITLE
Normal, Slow, Stall - Download Speed On Big File - File Sharing

### ISSUE THREAD

**Issue by verifiedsyckozis** [safing/spn#85]
**Pre-Submit Checklist**:

- Check applicable sources for existing issues:
  - [Github Issues](https://github.com/safing/spn/issues?q=is%3Aissue+label%3Abug)
  - [Broken Websites/Services](https://github.com/safing/spn/issues?q=is%3Aissue+label%3A%22website+compatibility%22)
  - [Network Status](https://github.com/safing/spn/issues?q=is%3Aissue+label%3A%22network%22%2C%22network+status%22)

**What do you want to report?**

**Pre-Submit Checklist**:

- Check applicable sources for existing issues:
  - [Github Issues](https://github.com/safing/spn/issues?q=is%3Aissue+label%3Abug)
  - [Broken Websites/Services](https://github.com/safing/spn/issues?q=is%3Aissue+label%3A%22website+compatibility%22)
  - [Network Status](https://github.com/safing/spn/issues?q=is%3Aissue+label%3A%22network%22%2C%22network+status%22)

**What happened**:

I was downloading something that I would normally do with a VPN and was getting "ok" speeds by comparison.  That I expected.  I'm not complaining in the slightest. 

**What did you expect to happen?**:

I expected nothing special to happen as spn being in Alpha.  This is just my reporting what happened.  The speed was at a normal 20 mbs.  Slowed to about 9 mbs, then 1-3 mbs, then about 90kbs, and then it stalled out.  It being the download.  I had to disconnect from spn and connect my VPN to finish the download, but what is good is that my surfing experience thus far is good.  So cheers to that.

The download was a .mp4

**How did you reproduce it?**:

I tried it on another file of the same type and similar size from the same source.

**Comment by dhaavi** [comment]
Hey @verifiedsyckozis, thanks a lot for the report - this is very helpful!

We are currently investigating an issue where browsing is very slow while downloading a big file. But from your description it seems that it is the other way around for you - correct?

I still think that the issues are connected. We suspect the source of the problems to be insufficient scheduling/prioritizing of different connections - where one connection would start to get all attention while others stall.

**Comment by verifiedsyckozis** [comment]
Yes, that seems to be the case.  I had and do have a fine browsing experience and streaming for that matter, but it's just those downloads weren't doing so well.

**Comment by verifiedsyckozis** [comment]
Sorry if this is a little much, but I've noticed that game streaming
services such as PS Premium, GeForce Now, Boosteroid, and Xcloud all need
to have the spn turned off to work.  That is if used through the apps.  I
haven't exactly used the browser versions yet, but I figured I'd also let
you know about this issue that I've run into, just in case it's something
that hasn't been encountered yet.
